include/sys/mimiker.h

```diff
@@ -138,7 +138,10 @@ extern char __etext[];
 extern char __boot[];
 extern char __eboot[];
 extern char __text[];
+extern char __etext[];
```

Comment:
There's a little mess in this file. Some external symbols are declared twice: before the `#ifdef _MACHDEP` directive and after.

Reply:
Cleaned up!